### ISSUE TITLE
The system cannot find the path or the file specified. 

### ISSUE THREAD

**Issue by horseman314159** [spyder-ide/spyder#21994]
### Issue Report Checklist

* [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [x] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [x] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [x] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [x] Tried basic troubleshooting (if a bug/error)
    * [x] Restarted Spyder
    * [x] Reset preferences with ``spyder --reset``
    * [ ] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [ ] Tried the other applicable steps from the Troubleshooting Guide
* [x] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
When I run codes in the editor or the console, for examples, print(1+1), the console would show the result and two sentences: The system cannot find the path specified. The system cannot find the file specified. No traceback errors and other things seem to be fine.

### What steps reproduce the problem?

1.  This occurs the first time the program is run after each restart of the kernel or spyder 

### Paste Traceback/Error Below (if applicable)

```python-traceback

PASTE TRACEBACK HERE

```
![ahoh](https://github.com/spyder-ide/spyder/assets/166992343/bd89d3c3-cdea-482d-a612-9e74f5da1223)

## Versions

* Spyder version: 5.5.4
* Python version: 3.10.14
* Qt version: 5.15.2
* PyQt version: 5.15.10
* Operating System name/version: windows 10

### Dependencies

```
PASTE DEPENDENCIES HERE
```
# Mandatory:
atomicwrites >=1.2.0             :  1.4.0 (OK)
chardet >=2.0.0                  :  4.0.0 (OK)
cloudpickle >=0.5.0              :  3.0.0 (OK)
cookiecutter >=1.6.0             :  2.6.0 (OK)
diff_match_patch >=20181111      :  20200713 (OK)
intervaltree >=3.0.2             :  3.1.0 (OK)
IPython >=8.13.0,<9.0.0,!=8.17.1 :  8.22.2 (OK)
jedi >=0.17.2,<0.20.0            :  0.19.1 (OK)
jellyfish >=0.7                  :  1.0.1 (OK)
jsonschema >=3.2.0               :  4.21.1 (OK)
keyring >=17.0.0                 :  24.3.1 (OK)
nbconvert >=4.0                  :  7.16.3 (OK)
numpydoc >=0.6.0                 :  1.5.0 (OK)
paramiko >=2.4.0                 :  2.8.1 (OK)
parso >=0.7.0,<0.9.0             :  0.8.3 (OK)
pexpect >=4.4.0                  :  4.8.0 (OK)
pickleshare >=0.4                :  0.7.5 (OK)
psutil >=5.3                     :  5.9.0 (OK)
pygments >=2.0                   :  2.17.2 (OK)
pylint >=3.1,<4                  :  3.1.0 (OK)
pylint_venv >=3.0.2              :  3.0.3 (OK)
pyls_spyder >=0.4.0              :  0.4.0 (OK)
pylsp >=1.11.0,<1.12.0           :  1.11.0 (OK)
pylsp_black >=2.0.0,<3.0.0       :  2.0.0 (OK)
qdarkstyle >=3.2.0,<3.3.0        :  3.2.3 (OK)
qstylizer >=0.2.2                :  0.2.2 (OK)
qtawesome >=1.3.1,<1.4.0         :  1.3.1 (OK)
qtconsole >=5.5.1,<5.6.0         :  5.5.1 (OK)
qtpy >=2.1.0                     :  2.4.1 (OK)
rtree >=0.9.7                    :  1.0.1 (OK)
setuptools >=49.6.0              :  69.2.0 (OK)
sphinx >=0.6.6                   :  5.0.2 (OK)
spyder_kernels >=2.5.1,<2.6.0    :  2.5.1 (OK)
textdistance >=4.2.0             :  4.2.1 (OK)
three_merge >=0.1.1              :  0.1.1 (OK)
watchdog >=0.10.3                :  2.1.6 (OK)
zmq >=24.0.0                     :  25.1.2 (OK)

# Optional:
cython >=0.21                    :  None (NOK)
matplotlib >=3.0.0               :  3.8.0 (OK)
numpy >=1.7                      :  1.26.4 (OK)
pandas >=1.1.1                   :  2.2.1 (OK)
scipy >=0.17.0                   :  1.13.0 (OK)
sympy >=0.7.3                    :  None (NOK)

**Comment by ccordoba12** [comment]
Hey @horseman314159, thanks for reporting. Please run the following command in the console

    %pwd

and post the result here. That would help us to better understand your problem.

**Comment by tony-pythony** [comment]
Here, the same behavior. On the first start of the debugger, I get the very same message displayed. But, only once: 
"System cannot find the path specified" not twice.
%pwd -> 'C:\\Users\\Anton\\Python\\notebook'


My dependencies:
# Mandatory:
atomicwrites >=1.2.0             :  1.4.0 (OK)
chardet >=2.0.0                  :  4.0.0 (OK)
cloudpickle >=0.5.0              :  2.2.1 (OK)
cookiecutter >=1.6.0             :  2.5.0 (OK)
diff_match_patch >=20181111      :  20200713 (OK)
intervaltree >=3.0.2             :  3.1.0 (OK)
IPython >=8.13.0,<9.0.0,!=8.17.1 :  8.22.2 (OK)
jedi >=0.17.2,<0.20.0            :  0.19.1 (OK)
jellyfish >=0.7                  :  1.0.1 (OK)
jsonschema >=3.2.0               :  4.19.2 (OK)
keyring >=17.0.0                 :  23.13.1 (OK)
nbconvert >=4.0                  :  7.10.0 (OK)
numpydoc >=0.6.0                 :  1.5.0 (OK)
paramiko >=2.4.0                 :  2.8.1 (OK)
parso >=0.7.0,<0.9.0             :  0.8.3 (OK)
pexpect >=4.4.0                  :  4.8.0 (OK)
pickleshare >=0.4                :  0.7.5 (OK)
psutil >=5.3                     :  5.9.0 (OK)
pygments >=2.0                   :  2.15.1 (OK)
pylint >=3.1,<4                  :  3.1.0 (OK)
pylint_venv >=3.0.2              :  3.0.3 (OK)
pyls_spyder >=0.4.0              :  0.4.0 (OK)
pylsp >=1.11.0,<1.12.0           :  1.11.0 (OK)
pylsp_black >=2.0.0,<3.0.0       :  2.0.0 (OK)
qdarkstyle >=3.2.0,<3.3.0        :  3.2.3 (OK)
qstylizer >=0.2.2                :  0.2.2 (OK)
qtawesome >=1.3.1,<1.4.0         :  1.3.1 (OK)
qtconsole >=5.5.1,<5.6.0         :  5.5.1 (OK)
qtpy >=2.1.0                     :  2.4.1 (OK)
rtree >=0.9.7                    :  1.0.1 (OK)
setuptools >=49.6.0              :  68.0.0 (OK)
sphinx >=0.6.6                   :  5.0.2 (OK)
spyder_kernels >=2.5.1,<2.6.0    :  2.5.1 (OK)
textdistance >=4.2.0             :  4.2.1 (OK)
three_merge >=0.1.1              :  0.1.1 (OK)
watchdog >=0.10.3                :  2.1.6 (OK)
zmq >=24.0.0                     :  25.1.2 (OK)

# Optional:
cython >=0.21                    :  None (NOK)
matplotlib >=3.0.0               :  3.8.3 (OK)
numpy >=1.7                      :  1.26.3 (OK)
pandas >=1.1.1                   :  2.1.4 (OK)
scipy >=0.17.0                   :  1.11.4 (OK)
sympy >=0.7.3                    :  1.12 (OK)

**Comment by horseman314159** [comment]
> Hey @horseman314159, thanks for reporting. Please run the following command in the console
> 
> ```
> %pwd
> ```
> 
> and post the result here. That would help us to better understand your problem.

C:\\Users\\my_user_name\\Desktop

**Comment by horseman314159** [comment]
>Hi fellows I occasionally fixed the issue by reinstalling spyder via Anaconda Navigator instead of conda install spyder. Hope that works for you

**Comment by tony-pythony** [comment]
> > Here, the same behavior. On the first start of the debugger, I get the very same message displayed
> 
> Could you post the message of the file you're trying to debug? Thanks!

I wanted to recreate the issue, but stupidly updated Spyder via Anaconda Navigator (*) (like horseman314159, too) and it seems to be fixed ??? When i do %pwd in the console, it shows now:
%pwd -> 'C:\\Users\\Anton\\Python\\notebook'

before it showed, obviously without escaping back-slash:
%pwd -> 'C:\Users\Anton\Python\notebook'

(*)
Some may be of interest information :
Did a fresh install of Anaconda Navigator & Spyder 5.5.1 using Navigator into base env after ditching my previous Conda setup with everything removed. 
Then I wanted to test Spyder 6.0 Alpha xx and installed it using conda in a newly created env "spyder-env" -> started it and played a little bit with it. Then closed Spyder 6 and started Spyder 5.5.1 for comparison again. After that, I wanted to try something in Spyder 6 - but it didn't start. I reset the spyder-env settings (shortcut in start menu) - but it still did not start. I was hoping, that Spyder 6 did not contaminate my Spyder 5 config directory :)
After that, I started to work with Spyder 5.5.1 again. At some point in time, Spyder 5 freeze and - it freeze so hard, that nothing worked anymore. I tried to kill python process in taskman. It tried - but after not forcefully killing it - nothing worked anymore. Windows (latest win10, 64bit) did not update display. Exiting other programs (firefox, taskman, explorer..) did not end their processes. Just a cold reboot helped here. (Really nothing worked. Waited for 10 mins -> cold reboot)
After reboot, Spyder 5 did not load, It stuck freeze upon initializing the plugins. No console, no variable manager, no toolbar. Although, the formerly opened files had been loaded showing the tabs to switch to the files. Here again, it freeze when I tried to click anywhere and/or close it for not doing anything -> reboot. I played this game 5 rounds then decided to reinstall Spyder. 

This time, i did create a new environment "spyder" for the new installation and used anaconda navigator to install Spyder 5.5.2 into the new environment. Did use it to reload my project, but after doing this I though "its stupid to have two Versions of Spyder" and deleted it again.
Then the "base" env Spyder did not work at all. Reset of the setting did not help. As before, it did not initialize the toolbar, the console or the pane with the plugins variable manager. Using Anaconda Navigator, I updated to Spyder 5.5.2 in the base environment. It did not start either. Resetting the Spyder configuration did not help. I reverted back to Spyder 5.5.1 with Anaconda Navigator. It did not start. 
Then, after googleing the issue, i reinstalled iPython console. It worked this after, finally.
Within this course of actions, something happened that lead to the "file not found" as described.

So, yesterday, I did update Spyder 5.5.1 to Spyder 5.5.4. The message of file not found is not displaying anymore.
the output of %pwd changed too. I don't know what happened in between.. :) I also can't reproduce it anymore. 
I'm very sorry for not being of help here anymore and will post an update, if it comes back again. 

(Thanks for the wonderful Spyder @all)

**Comment by ccordoba12** [comment]
Ok, thanks for letting us know about this no longer being an issue @horseman314159 and @tony-pythony.

I'll leave this one open for a bit longer to see if someone else has the same problem.

**Comment by ami-seculyze** [comment]
> I'll leave this one open for a bit longer to see if someone else has the same problem.

I do have the same issue (using Spyder 5.5.1). The most minimal example (e.g., 1+1) creates the exact same behaviour, i.e., it shows the sentence `The system cannot find the path specified.` but otherwise functions normally.

**Comment by ccordoba12** [comment]
@ami-seculyze, please also post the output of running `%pwd` in the console to see if that gives us more info.

**Comment by ccordoba12** [comment]
Closing due to lack of response.

**Comment by Eheran1** [comment]
This has been happening for many months to me, currently using spyder 5.5.4 and python 3.11.7:
`The system cannot find the path specified.`

```
%pwd
'C:\\Users\\[name*]\\.spyder-py3'
```
*No special characters, only ASCII abc... (characters 97 to 122), below 15 long

**Comment by ccordoba12** [comment]
@Eheran1, I think this is fixed in our latest version (6.0.3). So, please update and try again.

**Comment by Eheran1** [comment]
I am running Anaconda, so I have to wait for the update to trickle down. But thank you for letting me know.
